torch/_dynamo/allowed_functions.py

```diff
@@ -302,6 +302,7 @@ def _builtin_function_ids() -> Dict[int, str]:
     )
     rv.update({id(cast): "typing.cast"})
     rv[id(functools.reduce)] = "functools.reduce"
+    rv[id(polyfill.__wrapped_next)] = "polyfill.__wrapped_next"
```

Comment:
@ezyang what's a better way to do this? What I need is to have some custom metadata for `next` so that it can have custom behaviour when I call the `next` from polyfill.

Reply:
Honestly, if it were possible to handle `StopIteration` _only within polyfill_, our problems would be solved.

Reply:
Managed to achieve this, with a new `allow_stopiteration` while inlining generators. Hopefully it will be useful for other things too.